### DISCUSSION TITLE
How to add backgroud using TextEdit

### DISCUSSION THREAD

**Discussion by worldyuan** [slint-ui/slint#6712] [Q&A]
I want to set backgroud to transparent within TextEdit.

**Comment by ghost** [comment]
Hi,

thank you for your question. At the moment that is not directly possible on `TextEdit`. What you can do instead is to use [TextInput](https://docs.slint.dev/latest/docs/slint/src/language/builtins/elements#textinput) instead. For more details you can check the code of `TextEdit` it uses internal the `TextInput` https://github.com/slint-ui/slint/blob/master/internal/compiler/widgets/common/textedit-base.slint.

There is also a open issue to collect ideas and feedback on the topic how to support styling for the std-widget #5392.
